### DISCUSSION TITLE
how do i actually open or manage to config the hsmod?

### DISCUSSION THREAD

**Discussion by viniboi0** [Pik-4/HsMod#115] [Q&A]
because when i open hearthstone after installing everything correctly the log window opens, but i dont know where to locate the config file or open the bepinex window in hearthstone

**Comment by Pik-4** [comment]
Currently, due to the Hearthstone Unity update, the configuration management UI interface is invalid, and the hsmod configuration cannot be modified directly. You can manually modify bepinex/config/hsmod.cfg to change the configuration, or refer to #114
